flow/connectors/mysql/qvalue_convert.go

```diff
@@ -75,7 +75,7 @@ func qkindFromMysql(field *mysql.Field) (types.QValueKind, error) {
 	case mysql.MYSQL_TYPE_DATETIME2:
 		return types.QValueKindTimestamp, nil
 	case mysql.MYSQL_TYPE_TIME2:
-		return types.QValueKindTime, nil
+		return types.QValueKindTimestamp, nil
```

Comment:
nit: club these cases together

Reply:
case order is sorted by mysql type's byte value, but a more logical grouping seems fair